desktop/AutoStartManager.ts

```diff
@@ -62,10 +68,43 @@ class AutoStartManager {
     );
   }
 
-  static isEnabled = () => AutoStartManager.service.isEnabled();
+  static isEnabled = () => StoreService.get(IS_AUTO_START_ENABLED);
+
+  static restoreNativeAutoLaunchStatus = async () => {
+    try {
+      const isEnabled = await AutoStartManager.service.isEnabled();
+      const isConfigEnabled = StoreService.get(IS_AUTO_START_ENABLED);
+
+      // migrate isEnabled() native status to app config
+      // @TODO delete in the next releases this migration
```

Comment:
in the next releases, we will need just delete the migration flag from the config ( use method `remove` ) and invoke this code in this file